Aztec/Classes/TextKit/TextView.swift

```diff
@@ -428,115 +457,25 @@ open class TextView: UITextView {
     }
 
     open override func paste(_ sender: Any?) {
-        let pasteHandled = tryPastingURL() || tryPastingHTML() || tryPastingAttributedString() || tryPastingString()
-        
+
+        let pasteHandled = pasteboardDelegate.tryPastingURL(in: self)
```

Comment:
This is just a suggestion to go the extra mile:

Why not put this entire `try...` sequence into a main method in `pasteboardDelegate`?

This code would turn into something like:

```swift
let pasteHandled = pasteboardDelegate.paste(in: self) // or tryPaste() if you prefer
```

The advantage is that it would allow us to customize not only the logic inside each of those methods but also their priority ordering.

Reply:
This is a really great suggestion!!

Implemented in 30be23350587ac7ce6ab09f914b7c8106ddac805

I aligned the names in the protocol for consistency as well